data-integration/persist/model.bal

```diff
@@ -0,0 +1,25 @@
+import ballerina/persist as _;
+
+type Employee record {|
+    readonly int id;
+    string name;
+    int age;
+    string phone;
+    string email;
+    string department;
+    EmployeeTask[] employeeTask;
+|};
+
+type EmployeeTask record {|
+    readonly int taskId;
+    string taskName;
```

Comment:
id and name?

Reply:
Should I change the names to id and name, or only keep one of them instead of both?

Reply:
Should I change the names to id and name, or should I keep only one of them instead of both?

Reply:
I meant the former. Do we need both though? Can name be description instead?

Reply:
Shall I only keep the id then? It would be better I guess.

Reply:
Ah, we already have description. Was looking at it from the Conversation tab. +1 for just id.